node/src/service.rs

```diff
@@ -39,6 +39,8 @@ use crate::{
 pub struct PicassoExecutor;
 
 impl sc_executor::NativeExecutionDispatch for PicassoExecutor {
+	/// This should not be extended under any circumstances or we risk bricking our chain
```

Comment:
can you ELI5 when you say "this should not be extended?"

Reply:
This is where host functions can be extended, by supplying a tuple of them.